content/installation/java/configuration/YAML-properties-java.md

```diff
@@ -0,0 +1,203 @@
+<!--
+title: "YAML configuration properties for the Java agent "
+description: "Instructions for configuring Java agent properties via YAML file"
+tags: "installation java agent YAML configuration rules properties"
+-->
+
+Contrast supports YAML-based configuration for the Java agent. This allows you to store configuration on disk that you can override with environment variables or command line arguments.
+
+> **Note:** While property formatting in YAML configuration files is shared by all Contrast agents, each agent must use its specified file. 
+
+## Load Path
+
+Configuration values use the following order of precedence: 
+
+1. Corporate rule (e.g., expired license overrides `assess.enable`)
+1. System property value
+1. Environment variable value
+1. YAML configuration file value
+1. Contrast UI value
+1. Default value
+
+<!-- Other use and file location info. This will be done after CONTRAST-26125  -->
+
+Go to the [Java YAML Template](installation-netconfig.html#net-template) for fully formatted properties that you can copy and use in your own agent configuration files. 
```

Comment:
@kaitlynmortimer , I'm not sure how to comment on this template, but can you update it with the latest from my last PR, please? I'll send you the file privately. Thanks!

Reply:
The other agents should also make sure these Docs have the latest templates.